src/styles/components/cards.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  background-color: #fff;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+  }
+
+  &__name {
+    color: #060b35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+  }
+
+  &__code {
+    display: inline-block;
+    color: #616070;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 16px;
+    width: 166px;
+
+    &-count {
+      text-align: right;
+      font-size: 10px;
+      font-style: normal;
+      color: #060b35;
```

Comment:
Let's create variables for this and #616070 colors as you use them in a few places.